pubsub/kafka/metadata.yaml

```diff
@@ -170,6 +168,11 @@ authenticationProfiles:
         description: |
           AWS session token to use. A session token is only required if you are using\ntemporary security credentials.
         example: '"TOKEN"'
+      - name: awsIamRoleArn
+        type: string
+        description: |
```

Comment:
I think this description is OK, but I think the overarching description is bad and should be updated. This authType should support `creds` or `arn` as config, so I feel those details could be better described.

I feel like this description is not very good and could be improved: https://github.com/eunicecompra/components-contrib/pull/1/files#diff-7cbcd10673dca6e24c7064f4d1a23d9be6c876891114f24376b18e1015b61987L137

Reply:
Fair call. I will propose a change to the existing desc